README.md

```diff
@@ -275,6 +263,34 @@ More info [here](https://github.com/microsoft/vscode-eslint#mono-repository-setu
 
 Building a docker image, read the [docker doc](./docs/docker/docker.md).
 
+
+## 8. Releases, changelogs and versioning
+
+_Apps_ and _proxies_ use automatic semantic versions and are released using [Release Please](https://github.com/googleapis/release-please).
+
+> Release Please is a GitHub Action that automates releases for you. It will create a GitHub release and a GitHub Pull Request with a changelog based on conventional commits.
+
+Each time you merge a pull request, the release-please-action will create or update a "Release PR" with the changelog and the version bump related to the changes. To create a new release for an app, this release PR is merged, which creates a new release with release notes and a new tag. This tag will be picked by Azure pipeline and trigger a new deployment to staging. From there, the release needs to be manually released to production.
+
+### Conventional commits
+
+Use conventional commits to ensure that the changelogs are generated correctly.
```

Comment:
Maybe linking the "conventional commits" to https://www.conventionalcommits.org/ would make sense here.